spec/models/topic_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Topic, type: :model do
+  subject { create(:topic) }
+
+  describe 'validations' do
+    it { should validate_presence_of(:name) }
+    it { should validate_uniqueness_of(:name) }
+  end
+
+  describe '#create' do
+    it 'creates the topic' do
+      expect { subject }.to change { Topic.count }.by(1)
```

Comment:
It's possible to check the change from the initial value to the finish value. e.g.: 
`expect { subject }.to change { Topic.count }.from(0).to(1)`